test/extended/util/image/README.md

```diff
@@ -17,7 +17,11 @@ for easy mirroring by the `openshift-tests images` command.
    * Describe your use case in a PR to this file and have it approved
    * Define a standard CI build and publish the image to quay.io in the openshift namespace
```

Comment:
To the best of my understanding CI builds don't push to quay.io nor to the openshift namespace, so i'm confused by this.  

CI builds push to registry.svc.ci.openshift.org and generally in the ocp namespace (though certainly they can push to other namespaces)